apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -495,16 +551,24 @@ const DanceAiModal: React.FunctionComponent<DanceAiModalProps> = ({
   let aiBotHead = aiBotHeadNormal;
   let aiBotBody = aiBotBodyNormal;
   let previewAreaClass = undefined;
-  if (mode === Mode.GENERATING && generatingProgress.subStep >= 2) {
-    if (generatingProgress.step < BAD_GENERATED_RESULTS_COUNT) {
-      aiBotHead = aiBotHeadNo;
-      aiBotBody = aiBotBodyNo;
-      previewAreaClass = moduleStyles.previewAreaNo;
-    } else {
-      aiBotHead = aiBotHeadYes;
-      aiBotBody = aiBotBodyYes;
-      previewAreaClass = moduleStyles.previewAreaYes;
-    }
+  if (
+    mode === Mode.GENERATED ||
+    (mode === Mode.GENERATING &&
+      generatingProgress.step >= BAD_GENERATED_RESULTS_COUNT)
+  ) {
+    aiBotHead = aiBotHeadYes;
+    aiBotBody = aiBotBodyYes;
+    previewAreaClass = moduleStyles.previewAreaYes;
+  } else if (mode === Mode.GENERATING) {
+    const aiBotBodyThinkImages = [
```

Comment:
minor nit: move this array to the top of the file as it's a predefined constant array?

Reply:
Good call, updated.